src/script/conversation/MessageRepository.ts

```diff
@@ -1875,7 +1875,6 @@ export class MessageRepository {
     return this.sendAndInjectGenericCoreMessage(message, conversation, {
       ...options,
       skipInjection: true,
-      skipSelf: true, // We never want to forward calling messages to the self user
```

Comment:
This was a misunderstanding. The trick was to no inject in the selfClient (see `skipInjection`) not to prevent sending to the other devices.